lib/src/Uncertainty/Distribution/FisherSnedecorFactory.cxx

```diff
@@ -67,6 +81,10 @@ DistributionFactoryResult FisherSnedecorFactory::buildEstimator(const Sample & s
 
 FisherSnedecor FisherSnedecorFactory::buildAsFisherSnedecor(const Sample & sample) const
 {
+  // Use method of moments as starting point
+  FisherSnedecor estimatedMomentsDistribution = buildMethodOfMoments(sample);
```

Comment:
const FisherSnedecor

Reply:
Ok: I must focus on this point. This is fixed.